### PR TITLE
add link doc blog

### PR DESCRIPTION
**Pull request by annasmyk**

### CODE REVIEW

File: .github_jdemetra.Rproj
Comment (charphi) [inline .github_jdemetra.Rproj:1]: Do we need an Rproj in this repository?

File: .gitignore
Comment (charphi) [inline .gitignore:1]: Do we need an gitignore in this repository?

File: profile/README.html
Comment (charphi) [inline profile/README.html:1]: @annasmyk Is this file intended to be here ?

**Comment by TanguyBarthelemy** [comment]
See #2
